homeassistant/components/bryant_evolution/config_flow.py

```diff
@@ -74,15 +73,13 @@ async def async_step_reconfigure_confirm(
         if user_input is not None:
             system_zone = await _enumerate_sz(user_input[CONF_FILENAME])
             if len(system_zone) != 0:
-                our_entry = self._get_reconfigure_entry()
                 return self.async_update_reload_and_abort(
-                    entry=our_entry,
+                    self._get_reconfigure_entry(),
                     data={
                         CONF_FILENAME: user_input[CONF_FILENAME],
                         CONF_SYSTEM_ZONE: system_zone,
                     },
-                    unique_id=UNDEFINED,
-                    reason="reconfigured",
+                    reason="reconfigure_successful",
```

Comment:
`reconfigured` was not in `strings.json` - so I think it's better to migrate to use the common one.